docs/guides/validators/security_best_practices.md

```diff
@@ -332,18 +332,23 @@ sudo invoke-rc.d iptables-persistent save
 
 ### Rule for ssh connection
 
-:::danger
+::: tip
 
-This is just an **example** 
+We want to keep our SSH port open from the 192.168.1.3 network we blocked in the above case. That is we only want to allow those packets coming from 192.168.1.3 and which wants to go to the port 22.
```

Comment:
It's confusing. We didn't block anything yet. There is just fail2ban which is not block per se. It bans connections only after auth failure.

Reply:
fixed